### PR TITLE
[css-ui] Define  "native appearance" and rename control to widget

### PR DESCRIPTION
**Pull request by jugglinmike**
[css-ui] Define `control` #3990

### CODE REVIEW

File: css-ui-4/Overview.bs [hunk 1]
```diff
@@ -1785,13 +1785,16 @@ so this specification allows flexibility.
 </div>
 
 While the way most elements in a document look can be fully controlled by CSS,
-controls are typically rendered by UAs using native UI controls of the host operating system,
+<a>controls</a> are typically rendered by UAs using native UI controls of the host operating system,
 which can neither be replicated nor styled using CSS.
 
+The term <dfn>control</dfn> denotes the following [[HTML]] elements:
```
Comment (bkardell) [inline css-ui-4/Overview.bs]: Is it good to be quite this specific? What if things change? Are there disagreements even on some in terms of support? Could you say instead "Examples of controls in HTML are: ....."

Comment (jugglinmike) [reply css-ui-4/Overview.bs]: This definition, like the spec itself, is meant to be canonical. Ideally, if things change, they'd start by changing here. Even if we find ourselves updating the definition reactively, it seems to me that completeness is a goal of the standard. From that perspective, suggesting a partial list is less helpful than defining a complete set.

Comment (bkardell) [reply css-ui-4/Overview.bs]: is <summary>/<details>  not a control then?

Comment (jugglinmike) [reply css-ui-4/Overview.bs]: Good question! I'll defer to the editor on that.

Comment (frivoal) [reply css-ui-4/Overview.bs]: I'm thinking that maintaining this in a css spec is the wrong place. Tracking how the HTML spec changes will never be perfectly synchronized. Ideally, we'd just refer to a term defined in the HTML spec, and they'd update it when they add new elements. However, there doesn't seem to be one already.

Comment (SelenIT) [reply css-ui-4/Overview.bs]: > Ideally, we'd just refer to a term defined in the HTML spec

Doesn't the existing term [_widget_](https://html.spec.whatwg.org/multipage/rendering.html#widgets) satisfy this criteria? BTW, it already includes `<details>`.

**Comment by zcorpan** [comment]
I have changed this PR and https://github.com/whatwg/html/pull/4725 as per discussion in that PR. Please review.

**Comment by frivoal** [comment]
Not sure why the IPR checker is stuck, all contributors to this PR are good to go, so I'll merge anyway.

**Pull request merged**